eventlog/eventlog.go

```diff
@@ -58,19 +63,47 @@ func (l *EventLog) levelLog(level, message string, args ...any) {
 }
 
 func (l *EventLog) assembleArgs(args ...any) map[string]any {
+	if len(args) == 1 {
+		if aa, ok := args[0].([]any); ok {
+			return l.assembleArgs(aa...)
+		}
+	}
```

Comment:
This is 100% to deal with wrapped arrays coming from the Vue side. Took a while to figure this out.